cos_registration_server/devices/views.py

```diff
@@ -62,10 +62,7 @@ def __init__(
 
     cos_model_name = settings.COS_MODEL_NAME
 
-    grafana_main_link = (
-        f"{base_url}/{cos_model_name}-grafana/"
-        f"dashboards/"
-    )
+    grafana_main_link = f"{base_url}/{cos_model_name}-grafana/" f"dashboards/"
```

Comment:
this is weird no?

Reply:
fixed in 088a0785869d2e23d8ddfdf7f31589f8bc4327f4. I think it was a black auto format weirdness.